examples/grpc_spanner_example/src/test/java/com/example/grpc/service/ConnectServiceTest.java

```diff
@@ -0,0 +1,13 @@
+package com.example.grpc.service;
```

Comment:
The file is missing the license headers

Reply:
Remove this files since there is not unitests for this example